# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/senken/pathnames.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./config.h	2003-04-06 00:42:38.000000000 +0200
+++ ./config.h	2003-04-06 00:43:18.000000000 +0200
@@ -5,7 +5,7 @@
 #define HAVE_LC_MESSAGES 1
 #define HAVE_STPCPY 1
 #define HAVE_LIBSM 1
-#define PACKAGE_LOCALE_DIR "/usr/local/share/locale"
+#define PACKAGE_LOCALE_DIR "/usr/share/locale"
 /* #undef PACKAGE_DATA_DIR */
 /* #undef PACKAGE_SOURCE_DIR */
 
--- ./lib/utils.h	2003-03-26 08:35:38.000000000 +0100
+++ ./lib/utils.h	2003-04-06 00:43:54.000000000 +0200
@@ -57,8 +57,8 @@
 
 #define DAYS_MONTH 30
 
-#define IMG_DIRECTORY "/usr/local/share/senken/img/"
-#define GAMES_DIRECTORY "/usr/local/share/senken/games/"
+#define IMG_DIRECTORY "/usr/share/senken/img/"
+#define GAMES_DIRECTORY "/usr/share/senken/games/"
 
 #include "game.h"
 #include "map.h"
